### PR TITLE
Typo fix in signals.rst

### PR DESCRIPTION
**Pull request by enarve**
Just fixed a small typo that I noticed when reading "Using signals": 

> In this case you need **to to** connect the signal via code...

(Second "Note" in "Connecting a signal in the editor").

### CODE REVIEW

File: getting_started/step_by_step/signals.rst [hunk 1]
```diff
@@ -148,7 +148,7 @@ methods "_on_node_name_signal_name". Here, it'll be "_on_button_pressed".
 .. note::
 
     If you are using an external editor (such as VS Code) this
```
Comment (Calinou) [inline getting_started/step_by_step/signals.rst:150]:
```suggestion
    If you are using an external editor (such as VS Code), this
```

File: getting_started/step_by_step/signals.rst [hunk 2]
```diff
@@ -148,7 +148,7 @@ methods "_on_node_name_signal_name". Here, it'll be "_on_button_pressed".
...
-    automatic code generation might not work. In this case you need to to connect
+    automatic code generation might not work. In this case you need to connect
```
Comment (Calinou) [inline getting_started/step_by_step/signals.rst:151]:
```suggestion
    automatic code generation might not work. In this case, you need to connect
```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by mhilbrunner** [comment]
Cherrypicked to 4.3 in https://github.com/godotengine/godot-docs/pull/10346.

**Pull request merged**